### PR TITLE
feat: new rate backend, plugin hot swapping

### PR DESCRIPTION
**Pull request by kincaidoneil**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,21 +23,31 @@
...
     "dotenv": "^6.1.0",
     "envkey": "^1.2.2",
-    "ilp-connector": "^22.1.0",
-    "ilp-plugin-ethereum": "next",
-    "ilp-plugin-lightning": "next",
-    "ilp-plugin-mini-accounts": "^3.5.1",
+    "execa": "^1.0.0",
+    "ilp-connector": "^22.1.3",
+    "ilp-plugin-ethereum": "^3.0.0-beta.8",
+    "ilp-plugin-lightning": "^1.0.0-beta.14",
```
Comment (karzak) [inline package.json]:
```suggestion
    "ilp-plugin-lightning": "^1.0.0-beta.15",
```
Updated to use patched mini-accounts

File: run-connector.js [hunk 2]
```diff
@@ -1,33 +1,70 @@
...
+    }
+
+    watcher.on('ready', load)
+    watcher.on('change', reload)
+  }
+
+  // Start the conenctor
+  await listen()
+
+  const accountIds = [
```
Comment (karzak) [inline run-connector.js]: What about adding a peer or a new asym server without restarting? Could we maybe walk the `plugins` directory to recognize when new plugins are added?

Comment (berilac) [reply run-connector.js]: may be want to re-spell connector line 52? unless that was deliberate

Comment (berilac) [reply run-connector.js]: its funny when you can always find some one else's mistakes better than ones own :|

**Pull request merged**